src/cmd/honeycomb/main.go

```diff
@@ -50,12 +52,13 @@ func main() {
 			},
 			Logger: logger,
 		},
+		Cache:  cachingLocator,
```

Comment:
A little uncomfortable, but I can't see another way to do it, the docker locator needs to invalidate the cache.

Reply:
Yeah, originally I was going to leave it up to each locator to perform its own caching, but this means that static routes still need to be fully resolved on every request, so I settled on this approach.